src/Writer/Xml/XmlFormatter.php

```diff
@@ -72,18 +72,12 @@ public static function isEmptyValue($value): bool
             return true;
         }
 
-        if (is_string($value) && $value === '') {
-            return true;
-        }
-
-        return false;
+        return (bool) (is_string($value) && $value === '');
```

Comment:
Strange that `(bool)` is needed here...

Reply:
Yeah - you'd think there is no chance of it not being.

Reply:
@liamkeily Have removed it and it seems to pass